File: Array/Array Traversal/traversingArray.java
import java.util.*;

public class traversingArray 
{
    public static void main(String[] args)
    {
        try(Scanner sc = new Scanner(System.in))
        {
            System.out.print("Enter the size of the array: ");
            int arraySize = sc.nextInt();

            int array[] = new int[arraySize];

            System.out.println("Enter the " + arraySize + " elements of array: ");

            for(int i=0; i<arraySize; i++)
            {
                System.out.print("Element ["+ i +"]: ");
                array[i] = sc.nextInt();
            }

            System.out.println("The elements of array are: ");

            System.out.print(array[0]);
            for(int i=1; i<arraySize; i++)
            {
                System.out.print(", "+array[i]);
            }
        }    
    }    
}

File: Array/Array Traversal/traversing_array.py
# scanning the size of the array
size = int(input("Enter the number of element to store in array: "))

# declaring the empty array
array_1 = []

# scanning the elements of the array
print(f"Enter the {size} elements of the array: ")
for i in range(size):
    element = int(input(f"Element {i}: "))
    array_1.append(element)

# printing the array
print("Array:", end=" ")
for i in range(size):
    print(array_1[i], end=" ")
